### DISCUSSION TITLE
Port direct SMTP sending support

### DISCUSSION THREAD

**Discussion by wleightond** [thinkst/canarytokens#274] [Feature Requests]
Canarytokens v2 supported a direct SMTP email channel. Currently v3 only supports Mailgun and SendGrid as mail providers, so we should port the direct SMTP functionality to v3 as well.

**Comment by emmanuel-thinkst** [comment]
Hey @wleightond 

That would be fantastic, and hopefully we can better handle the `SMTP` issue that came up in `canarytokens` v2. (https://github.com/thinkst/canarytokens-docker/issues/134)

Thanks
